src/AppKernel.php

```diff
@@ -179,6 +180,11 @@ public function loadRoutes(LoaderInterface $loader)
         return $routes->build();
     }
 
+    public function setCachePrefix($cachePrefix)
```

Comment:
Why is this needed?

Reply:
Before it was just used the first argument from the constructor, I think we could also remove this.